watchfrr/watchfrr.c

```diff
@@ -961,6 +964,53 @@ static char *translate_blanks(const char *cmd, const char *blankstr)
 	return res;
 }
 
+static void watchfrr_init(int argc, char **argv)
+{
+	const char *special = "zebra";
+	int i;
+	struct daemon **add = &gs.daemons, *dmn;
```

Comment:
I'm probably missing something, so for my own education, isn't this equivalent to just
```
struct daemon **add = &gs.daemons;
```

Why the dereference of `dmn` after the sequence point?

Reply:
That's a declaring-multiple-variables comma, not a comma operator

Reply:
(switched the order around to make that more obvious)

Reply:
ack